en/includes/guides/users/attributes/manage-attributes.md

```diff
@@ -53,7 +53,12 @@ Apart from the default attributes, you may define your own custom attributes by
 
 4. Click **Finish**.
 
-## Update attributes
+{% if product_name == "WSO2 Identity Server" %}
```

Comment:
don't we need to mention the version as well?

Reply:
Fixed.